cmd/nitro/init.go

```diff
@@ -405,6 +405,33 @@ func databaseIsEmpty(db ethdb.Database) bool {
 	return !it.Next()
 }
 
+func extractSnapshot(archive string, location string, importWasm bool) error {
+	isWasmDB := regexp.MustCompile("^(./)?wasm")
```

Comment:
This misses `././wasm`. If geth's regex engine supports it, I think `(^|/|\\)?wasm` would work well, meaning that it'd match "wasm" either at the start of the string or prefixed by a slash or backslash for Windows (make sure the backslash is properly escaped, it might need double escaped since it's in a string). We should add a test case for the regex as well, testing that `wasm`, `./wasm`, `././wasm`, `nitro/../wasm`, `.\wasm`, `.\.\wasm`, `wasm\0test`, `./wasm\0test` all match (where `\0` is the null byte) and e.g. `not-wasm` and `l2chaindata/example` don't match.

Reply:
I should have followed my intuition not to use a regex in the first place. I replaced it with filepath functions, and it should cover all these cases, including Windows.